packages/core/test/flow/combinator/merge.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import { merge, now, runEffects } from '../../../src/index'
+import { type Stream } from '@most/types'
+import { newDefaultScheduler } from '@most/scheduler'
+
+const s1 = now(123)
+const s2 = now('a')
+const s = merge(s1, s2)
+
+runEffects(s, newDefaultScheduler())
```

Comment:
Strange test, but I guess it’s a type assertion test in the IDE. I’m not sure 🤔 it is okay, or do the node tests also check types?

Reply:
Ah, I see the path is `test/flow`

Reply:
Yeah, see the readme in test/flow.  It'd probably be better to try to get typechecking working in merge's regular unit tests.  I tried briefly, but something must not be right in our flow config, since I couldn't get type checking to fail when it should :/